src/FsAutoComplete.Core/Commands.fs

```diff
@@ -1120,3 +1120,28 @@ type Commands (serialize : Serializer, backgroundServiceEnabled) =
     }
 
     member x.GetChecker () = checker.GetFSharpChecker()
+
+    member x.ScopesForFile (file: string) = async {
+        let rec blockForCheckResults file opts = async {
```

Comment:
this was an experiment to see if I could loop until I got recent results for a file, but it didn't work. needs to be deleted.